### PR TITLE
Update to latest snapshot + Work on GUI and Worldgen

### PR DESCRIPTION
**Pull request by ExE-Boss**
Depends on: NOVA-Team/NOVA-Core#229, NOVA-Team/NOVA-Core#231, NOVA-Team/NOVA-Core#232, NOVA-Team/NOVA-Worldgen#1 and NOVA-Team/NOVA-GUI#8.

---

P.S.: commits: 0ec4420, de2921a and b3becb7 are the exact same commit, just some bug with GitHub for Windows caused them to be committed thrice.

### CODE REVIEW

File: block/src/main/java/nova/sample/block/NovaBlock.java [hunk 1]
```diff
@@ -47,10 +47,10 @@
...
 
 	public NovaBlock(BlockManager blockManager,
-					 ItemManager itemManager,
-					 RenderManager renderManager,
-					 NetworkManager networkManager,
-					 RecipeManager recipeManager) {
+	                 ItemManager itemManager,
+	                 RenderManager renderManager,
+	                 NetworkManager networkManager,
+	                 RecipeManager recipeManager) {
```
Comment (RX14) [inline block/src/main/java/nova/sample/block/NovaBlock.java:53]: Have you imported the NOVA standard format on IDEA? Make sure you use it.

Comment (ExE-Boss) [reply block/src/main/java/nova/sample/block/NovaBlock.java:53]: The spaces being used as leading whitespace in constructors was already done like that in NovaWorldgen, I just manually copied it to the other mod classes.

Comment (RX14) [reply block/src/main/java/nova/sample/block/NovaBlock.java:53]: Actually, this probably is correct. Been a while since I used the formatter.

**Comment by ExE-Boss** [comment]
Moved to #32

**Pull request closed without merging**